src/main/kotlin/io/github/gradlenexus/publishplugin/NexusPublishPlugin.kt

```diff
@@ -76,6 +77,14 @@ class NexusPublishPlugin : Plugin<Project> {
                     "release${capitalizedName}StagingRepository", rootProject.objects, extension, repository, registry)
             val closeAndReleaseTask = rootProject.tasks.register<Task>(
                     "closeAndRelease${capitalizedName}StagingRepository")
+            getStagingProfileTask {
+                description = "Gets and displays a staging profile id for a given repository and package group. This is a diagnostic task to get the value and put it into the NexusRepository configuration closure as stagingProfileId."
+                group = PublishingPlugin.PUBLISH_TASK_GROUP
```

Comment:
I'm not sure it is worth displaying the task in the common output for `./gradlew tasks`.
Don't get me wrong, but I believe very few end-users would call `./gradlew get... StagingProfile`.

Please keep `group=null` so the task does not appear in `gradlew tasks` output

Reply:
Assuming its one-time per project (or even a group of projects) role, I agree that keeping it mentioned only in the documentation (in README) would be enough.

Reply:
Definitely happy to defer to you all on this one, but for what it's worth the first thing I did when trying to figure out how to get the staging profile with this new plugin was running `./gradlew tasks` and checking the output for a similar task